packages/braid-design-system/src/lib/components/Toggle/Toggle.docs.tsx

```diff
@@ -82,17 +86,60 @@ const docs: ComponentDocs = {
               on={getState('toggleLeading')}
               onChange={() => toggleState('toggleLeading')}
               togglePosition="leading"
+              bleedY
             />
             <Toggle
               label="Trailing"
               id={`${id}_trailing`}
               on={getState('toggleTrailing')}
               onChange={() => toggleState('toggleTrailing')}
               togglePosition="trailing"
+              bleedY
             />
           </Stack>,
         ),
     },
+    {
+      label: 'Vertical bleed',
+      description: (
+        <Fragment>
+          <Text>
+            With the <Strong>bleedY</Strong> prop, you can remove excess
+            vertical space created by the toggle input.
+          </Text>
+          <Text>
+            For example, you can include a toggle in a{' '}
+            <TextLink href="/components/Stack">Stack</TextLink> with even
+            vertical spacing. If you disable the <Strong>bleedY</Strong> prop
+            below, the toggle input will introduce unwanted space above and
+            below the toggle label.
+          </Text>
```

Comment:
I've riffed on this section a bit, thoughts?

![image](https://github.com/seek-oss/braid-design-system/assets/912060/de1e99f0-70bd-4e21-b0f8-39094b77335c)

```suggestion
          <Text>
            The <Strong>bleedY</Strong> prop removes the excess vertical space
            created by the toggle input — ensuring it’s only as tall as the{' '}
            provided label text.
          </Text>
          <Text>
            This better aligns with the{' '}
            <TextLink href="/foundations/layout">layout principles</TextLink> of
            the system, enabling the surrounding white space to be controlled by
            the parent layout component, e.g.{' '}
            <TextLink href="/components/Stack">Stack</TextLink>.
          </Text>
          <Alert tone="caution">
            <Text>
              In the future this will become the default behaviour. Migrating
              layouts to work with this option now is recommended.
            </Text>
          </Alert>
          <Text>
            In the following example, a{' '}
            <TextLink href="/components/Stack">Stack</TextLink> controls the
            space between child elements. Switching the toggle off demonstrates
            the introduction of unwanted space above and below the label.
          </Text>
```

Reply:
Just changed a couple minor things. I mentioned it will become "standard" behaviour rather than "default" because we won't have no bleed as an option at all. What do you think?

Reply:
Yep love it